style/ruby/README.md

```diff
@@ -38,6 +38,9 @@ Ruby
 * Use heredocs for multi-line strings.
 * Prefer `protected` over `private` for non-public `attr_reader`s, `attr_writer`s,
   and `attr_accessor`s.
+* Break up method definitions over 80 characters long by
```

Comment:
I wonder if this can just be extrapolated based on `Use a trailing comma after each item in a multi-line list, including the last item.` aka - it seems similar enough that you might just know what to do in this case based on the general rule...but I am not 100% sure. interested in what others think